cypress/e2e/shared/helpBar.test.ts

```diff
@@ -0,0 +1,149 @@
+describe('Help bar menu sub nav links', () => {
+  beforeEach(() => {
+    cy.flush()
+    cy.signin()
+    cy.visit('/')
+    cy.setFeatureFlags({
+      helpBar: true,
+    })
+  })
+
+  it('checks if help bar InfluxDB documentation link is correct', () => {
+    cy.getByTestID('tree-nav--sub-menu')
+      .get('#documentation')
+      .within(() => {
+        cy.get('a').should($a => {
+          expect($a.attr('href'), 'href').to.equal(
+            'https://docs.influxdata.com/'
+          )
```

Comment:
initially, i had a code here that changed the anchor tag's target attribute from `_blank` to `_self`  to open the link in the same tab, but that caused a cross-origin error. The goal of this test here is to make sure the href is correct. So, I don't see the point behind clicking the link and opening the page. If the href is what we expect, we can safely assume the user will be able to access the page.